applications/crossbar/src/modules/cb_att_handlers_errors.erl

```diff
@@ -0,0 +1,133 @@
+%%%-----------------------------------------------------------------------------
+%%% @copyright (C) 2018-, 2600Hz
+%%% GET     /:api_version/accounts/:account_id/att_handlers_errors[/handler/handler-id]
```

Comment:
Either write a proper sentence (starting with `%%% @doc`) or remove this line and just add (`%%% @doc`)

Reply:
`%%% @doc API to retrieve attachment handlers errors.`